Neuron/Sections/Wallet/NFTDetailViewController.swift

```diff
@@ -43,8 +44,8 @@ class NFTDetailViewController: UICollectionViewController {
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if segue.identifier == "pushToDescription" {
-            let commonWebViewController = segue.destination as? CommonWebViewController
-            commonWebViewController?.url = URL(string: assetsModel.external_link ?? "https://www.nervos.org")
+            let safariController = SFSafariViewController(url: URL(string: assetsModel.external_link ?? "https://www.nervos.org")!)
```

Comment:
Either define the `SFSafariViewController` from storyboard for the segue, or use a normal action and presenting the vc here. (This means a storyboard segue should specify both the target view controller and how it's presented)

Reply:
ok, it's a bug, thanks.